src/app/views/initial-setup/initial-setup-tips/initial-setup-tips.component.html

```diff
@@ -1,28 +1,29 @@
 <div class="initial-setup-tips">
   <app-close-icon></app-close-icon>
-    <div *ngIf="os==='iOS'">
-    <h5>Create Shortcut:</h5>
-    <h5>Step 1:</h5>
+  <h5>Great you made it!</h5>
+  <h6>Here is a tip for creating a shortcut:</h6>
+  <div>
+    <h6>Step 1:</h6>
     <div class="initial-setup-tips__icon"> 
-      <img src="assets/share_apple.png" class="initial-setup-tips__image">
-    </div>
-    <h5 class="initial-setup-tips__text">
-      Step 2:
-    </h5>
-    <div class="initial-setup-tips__icon">
-      <img src="assets/add_apple.png" class="initial-setup-tips__image">
-    </div>
-  </div>
-  <div *ngIf="os!=='iOS'">
-    <h5>Create Shortcut:</h5>
-    <h5>Step 1:</h5>
+      <div *ngIf="os==='iOS'">
+        <img src="assets/share_apple.png" class="initial-setup-tips__image">
+      </div>
+      <div *ngIf="os==='Samsung'">
+        <img src="assets/add_samsung.png" class="initial-setup-tips__image">
+      </div>
+      <div *ngIf="os!=='iOS' && os!=='Samsung'">
+        <img src="assets/share_android.png" class="initial-setup-tips__image">
+      </div>
+    </div>  
+    <h6 class="initial-setup-tips__text">Step 2:</h6>
     <div class="initial-setup-tips__icon"> 
-      <img src="assets/share_android.png" class="initial-setup-tips__image">
-    </div>
-    <h5>Step 2:</h5>
-    <div class="initial-setup-tips__icon">
-      <img src="assets/add_android.png" class="initial-setup-tips__image initial-setup-tips__image--small">
-    </div>
+      <div *ngIf="os==='iOS'">
+        <img src="assets/add_apple.png" class="initial-setup-tips__image">
+      </div>
+      <div *ngIf="os!=='iOS' && os!=='Samsung'">
+        <img src="assets/add_android.png" class="initial-setup-tips__image--small">
+      </div>
```

Comment:
with this logic, a samsung user would see:

```
Step 1:
<image>
Step 2:
<empty>
```

Reply:
fixed. put another *ngIf before step 2